### ISSUE TITLE
Add missing copyright notices

### ISSUE THREAD

**Issue by elbaloo** [nunit/nunit#840]
While working on #824 I noticed some files likes like:
- [TestSuiteResult.cs](https://github.com/nunit/nunit/blob/master/src/NUnitFramework/framework/Internal/Results/TestSuiteResult.cs),
- [IXmlNodeBuilder.cs](https://github.com/nunit/nunit/blob/master/src/NUnitFramework/framework/Interfaces/IXmlNodeBuilder.cs) (and some others)

that don't have a copyright notice at the top.
Should I go ahead and add the notice to these?
If so, what year should I put in the first line?

**Comment by rprouse** [comment]
Sure, go ahead Use 2015 and Charlie's name.

**Comment by CharliePoole** [comment]
I'm surprised you found so many missing, but maybe I shouldn't be. For NUnit 2.x, we used to run a script over all the source to identify those files without a license and copyright. It was written in PHP. 

When we started NUnit 3.0, I didn't want to use PHP as a scripting language any longer. We had a few initial discussions about what to use but it never got resolved. Maybe I should just pick something and use it to rewrite the copyrigtht checker. :-)
